### DISCUSSION TITLE
How to add Jekyll Image slider in this template?

### DISCUSSION THREAD

**Discussion by mihir022** [alshedivat/al-folio#1813] [Q&A] [answered]
@pourmand1376 @alshedivat @george-gca @rohandebsarkar 
I am using this template to make a website for our lab. I want to add an Image Slider in the About page just below the description. What process or required steps should I follow to add js enabled image slider? Can you tell me what changes should I do in which particular files to reflect the slider on the About page?
Here is demo of slider [Image Slider](https://jekylltools.github.io/jekyll-ideal-image-slider-include/examples/).

Will you please help me add this to my website? I am trying to follow steps from this given example but it is not reflecting.

**Comment by george-gca** [comment, marked as answer]
Are you following the steps given in their repo [README](https://github.com/jekylltools/jekyll-ideal-image-slider-include)?

PS: it might simply not work, since the Jekyll Image slider repo haven't been updated in years, and as is stated in the information on the right:

> This repository is archived. Issues are disabled. Pull requests will be ignored.

But if you want to try it, I would recommend following the steps in their README and also in the link you provided. On our side you would probably need to modify only the files `_pages/about.md` and `_layouts/default.html`.
